Tasks/ArchiveFilesV2/archivefiles.ts

```diff
@@ -94,6 +100,13 @@ function zipArchive(archive: string, files: string[]) {
     }
     var zip = tl.createToolRunner(xpZipLocation);
     zip.arg('-r');
+    // Verbose gets priority over quiet
+    if (verbose) {
```

Comment:
Is the default that it does verbose? Looks like the initial bug report said that. Do we want an enum of Default, Verbose, Quiet? It seems like the report said it starts off by default as logging too much?

Reply:
So the issue is that the default is different across tools.
- If you use tar or 7-zip, the default is quiet (or as close as they have to quiet)
- If you use zip, the default is neither quiet nor verbose (in between)

Quiet really only has meaning in the context of zip since the others don't have a quiet option - the initial bug was using zip. Also, for clarity, these changes won't change the default behavior unless the verbose or quiet flags are supplied.

Reply:
Discussed offline. Everything looks good.